README.md

```diff
@@ -54,43 +54,47 @@ It doesn't come out of the box with @fs/eslint-config-frontier-react, so you'll
+### As of v11 of this repository, we no longer keep codeclimate compatability. [See this Architectural Decision why.](https://www.familysearch.org/frontier/docs/architectural-decisions/020-eslint-codeclimate-compatability)
 
 ### For People using v10 and before and still want eslint in codeclimate we left the following instructions
 
 It is important to note that in order for CodeClimate to use this custom config, we have to work around their limitations a bit.
 
-1.  Add a prepare section to your .codeclimate.yml that will download this eslint-config file. [Prepare Docs](https://docs.codeclimate.com/docs/configuring-the-prepare-step)
-    - It should look like this
-      ```yaml
-      prepare:
-        fetch:
-          - url: 'https://raw.githubusercontent.com/fs-webdev/eslint-config-frontier-react/master/codeclimateEslintRulesv10.js'
-            path: 'eslint-config-frontier-react.js'
-      ```
-    - If you also opt into the jsdoc plugin we provide, you'll also need to make a prepare statement for that file.
-    - WARNING: Starting in version 4 of this repo, there is a different codeclimateEslintRules file for every major version
-    that is released. If you are using or upgrading to v8 of this repo, you'll need to change the fetched url file to 
-    `codeclimateEslintRulesv10.js`. Notice the "v10" at the end of the filename. When v11 is released, you'll also need
-    to change the filename to v11, etc.
-
-2.  Make a new eslintrc file for codeclimate to use (that way it can point to the `eslint-config-frontier-react.js` file that codeclimate will prepare in Step 1.)
-
-    1.  Copy your existing local eslintrc file and rename the copy to `.codeclimate.eslintrc.js` (or .json or .yml if you are using those filetypes)
-    2.  Change the `"extends": ["frontier"]` statement to point to the prepared file from Step 1. `"extends": ["./eslint-config-frontier-react.js"]`
-        (only do this in `.codeclimate.eslintrc.js` file, not your normal eslintrc)
-        - If you also opt into the jsdoc plugin we provide, you'll also need to add that downloaded file to the extends array
-
-3.  Tweak your .codeclimate.yml eslint section to point to the .codeclimate.eslintrc file instead of your default local eslintrc file
-    - Your plugin section in your .codeclimate.yml may be larger and more complicated, but the `config:` part should point to the new eslintrc file you made in step 2.
-    ```yaml
-    plugins:
-      eslint:
-        channel: eslint-5
-        config: # <- This line and the line below it are the important lines to add/tweak
-          config: .codeclimate.eslintrc.js # <- this line and the line above it are the important lines to add/tweak
-          extensions:
-            - .js
-            - .html
-        enabled: true
-    ```
+1. Add a prepare section to your .codeclimate.yml that will download this eslint-config file. [Prepare Docs](https://docs.codeclimate.com/docs/configuring-the-prepare-step)
```

Comment:
most of this updated because of the markdownlint I have. I did remove the line `When v11 is released, you'll also need
    to change the filename to v11, etc.`